src/components/Button/__tests__/Button.visual.test.tsx

```diff
@@ -2,9 +2,22 @@ import React from 'react';
 
 import {test} from '~playwright/core';
 
+import {createRegressionScenarios} from '../../../stories/tests-factory/create-regression-scenarios';
+import {createSmokeScenarios} from '../../../stories/tests-factory/create-smoke-scenarios';
+import {Button} from '../Button';
+
+import {
+    defaultProps,
+    disabledCases,
+    loadingCases,
+    pinsCases,
+    selectedCases,
+    sizeCases,
+    viewsCases,
+} from './cases';
 import {ButtonStories, CustomIconSizeButton} from './helpersPlaywright';
 
-test.describe('Button', () => {
+test.describe('Button', {tag: '@Button'}, () => {
```

Comment:
added a tag for the component so that you can run tests for one or more components